activestorage/lib/active_storage/engine.rb

```diff
@@ -112,7 +112,8 @@ class Engine < Rails::Engine # :nodoc:
           when /image_processing/
             ActiveStorage.logger.warn <<~WARNING.squish
               Generating image variants require the image_processing gem.
-              Please add `gem 'image_processing', '~> 1.2'` to your Gemfile.
+              Please add `gem "image_processing", "~> 1.2"` to your Gemfile
```

Comment:
Small cosmetic change with the quotes here but it matches what is generated in the gemfile and the error message in `image_processing_transformer.rb`